README.md

```diff
@@ -640,6 +661,125 @@ or
 ecs-cli compose --ecs-params my-ecs-params.yml service up --launch-type FARGATE
 ```
 
+#### Using Route53 Service Discovery
+
+With the ECS CLI, you can create a Service that uses [Route53 auto naming for service discovery](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html). Service Discovery requires a Service Discovery Service, and a DNS Namespace. When you enable Service Discovery with the ECS CLI, a new Service Discovery Service is always created using CloudFormation. For the DNS Namespace, you have the option of using an existing public or private DNS Namespace, or letting the ECS CLI create a private DNS Namespace for you using CloudFormation. Creation of a Public DNS Namespaces is not supported with the ECS CLI. Please note that only a single DNS Namespace may be used with Service Discovery.
+
+##### Enabling Service Discovery
+
+###### Specifying Values
+
+There are many input parameters for Service Discovery, however, for most of them, the ECS CLI will choose reasonable defaults. Here are the default values and explanations listed with the ECS Params input schema:
+
+```
+version: 1
+run_params:
+  service_discovery:
+    // container name and port are the container that will be used for Service Discovery
+    container_name: string            // No Default, required if using SRV records
+    container_port: string            // No Default, required if using SRV records
+    private_dns_namespace:
+      vpc: string                     // Required if ID is not specified
+      id: string                      // Allows you to specify an existing namespace by ID
+      name: string                    // DNS name for the namespace, if it already exists, the CLI will use, if not, the CLI creates one
+      description: string             // Only used if the namespace does not yet exist, Default = Created by the Amazon ECS CLI
+    public_dns_namespace:
+      id: ns-C0VF3F3                  // Specify an existing public namespace by ID
+      name: amazon.com                // Or specify an existing public namespace by Name
+    service_discovery_service:
+      name: string                    // Default = Name of the your ECS Service
+      description: string             // Default = Created by the Amazon ECS CLI
+      dns_config:
+        type: string                  // Valid values: A or SRV, SRV is default/required with network modes bridge and host, A is the default for awsvpc
```

Comment:
Period instead of first comma.

Also maybe rephrase the second half to be something like "`SRV` is required/the default when using `bridge` or `host` network mode. `A` is the default for `awsvpc` network mode."